src/components/CText/CText.js

```diff
@@ -64,7 +64,7 @@ export default {
       },
     };
 
-    if (isNil(this.textValue) || isEmpty(this.textValue)) {
+    if (isNil(this.textValue) || this.textValue === '') {
```

Comment:
What are the expected values for `this.textValue`? Is it just a string, empty string and some falsy value value? Could just a check for falsy values work? 
```
if (!this.textValue) { ... }
```
Anyway, if you're expecting an object for some reason and hence using `isEmpty`, you don't need `isNil` since you've covered those cases with `isEmpty`.

Reply:
Nop, no need for object
Null is default value, same is for almost every component but when you add some value and remove it (inside builder) option type _input_ is returning empty string, or "". 
That would be it...

Reply:
Maybe, there was some other value also. I'm not sure at the moment.

Reply:
Anyway, if just `null` or `''` are pre-conditions for displaying the placeholder, you can just use an ordinary falsy check.

On other hand, if something like an object can be expected, you don't need both `isNil` and `isEmpty` in an OR check since `isEmpty` covers for both.

Reply:
I'll change it to "falsy check". I can't recall what was the reason for null check, if there was one I guess will find out soon :)